pycbc/transforms.py

```diff
@@ -314,7 +314,7 @@ def inverse_transform(self, maps):
             A dict with key as parameter name and value as numpy.array or float
             of transformed values.
         """
-        sx, sy, sz = self._outputs
+        sx, sy, sz = self._inputs
```

Comment:
This looks wrong. This is the inverse transform, so the inputs to this function are the `_outputs` of the transform. Why are you changing it?

Reply:
This is actually broken. I've reverted this line and changed the other function. You can't just copy the inputs, outputs, and transform functions because they explicitly call ``self._inputs`` and ``self._outputs``. They won't map properly. I added a ``transform`` and ``inverse_transform`` to the other class to fix this.